config/routes.rb

```diff
@@ -1,5 +1,10 @@
 Rails.application.routes.draw do
+  root 'pages#index'
+  
   devise_for :users
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  root to: 'welcome#index'
+
+  resources :help, only: [:index]
+  resources :contacts, only: [:index]
+  resources :about, only: [:index]
+  resources :tutor, only: [:index], path: 'become a tutor'
```

Comment:
There is no reason in creating separate controllers and view direcrories for static pages, you can specify route like
`get ':page_name', controller: :pages, action: :index, constraints: {page_name: /(about|contacts|become_a_tutor|help)/}`

and then in `app/views/pages/index.html.slim` there will be something like:
`= render[:page_name].presence || 'landing'`

and move content for that pages to `app/views/pages/_(landing|about|help|contacts|become_a_tutor).html.slim`